packages/amplication-client/src/Entity/EntityFieldProperty.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Icon } from "@amplication/design-system";
+import "./EntityFieldProperty.scss";
+type Props = {
+  property: string;
```

Comment:
change property prop name to iconTitle or fieldPropertyTitle

Reply:
Removed this Props type instead given the type within `React.FC<>`✅